base/migrator/migrator.Job.yaml

```diff
@@ -0,0 +1,56 @@
+---
```

Comment:
You'll need to add this file to the kustomization bases list [here](https://sourcegraph.com/github.com/sourcegraph/deploy-sourcegraph/-/blob/overlays/bases/deployments/kustomization.yaml).

There might be some additions to the kustomize overlays too - the only one that stands out is on the non-privileged overlay, something similar to https://sourcegraph.com/github.com/sourcegraph/deploy-sourcegraph/-/blob/overlays/non-privileged/frontend/sourcegraph-frontend.Deployment.yaml so this can run as non-root.